pipelines/ocw/docker-image-pipeline.yml

```diff
@@ -0,0 +1,33 @@
+---
+resources:
+    - name: ol-infrastructure
+      type: git
+      source:
+          uri: https://github.com/mitodl/ol-infrastructure.git
+          branch: ((ol-infrastructure-git-ref))
+    - name: node-image
+      type: docker-image
+      check_every: 1h
+      source:
+          repository: node
+          tag: '14'
+    - name: ocw-course-publisher-image
+      type: docker-image
+      source:
+          username: ((dockerhub.username))
+          password: ((dockerhub.password))
+          repository: mitodl/ocw-course-publisher
+          tag: latest
+
+
+jobs:
+    - name: publish
+      public: true
+      serial: true
+      plan:
+          - get: node-image
+            trigger: true
+          - get: ol-infrastructure
```

Comment:
Should probably add a trigger for this as well with a path glob to match the Dockerfile that it's relying on.

Reply:
👍

Reply:
adf00b6d should have changes to the Dockerfile trigger builds.